docs/ref/templates/builtins.txt

```diff
@@ -261,8 +261,8 @@ Sample usage::
 ``firstof``
 -----------
 
-Outputs the first argument variable that is not ``False``. Outputs nothing if
-all the passed variables are ``False``.
+Outputs the first argument variable that is not ``Falsy`` value. Outputs nothing if
+all the passed variables are ``Falsy``.
```

Comment:
I think we shouldn't use `falsey` here. I would prefer to use more descriptive sentence (like in the `if` docs), i.e.
```
Outputs the first argument variable that is not "false" (i.e. doesn't exist, is
empty, or is a false boolean value). Outputs nothing if all the passed
variables are "false".
```

Reply:
Sure, picked it up the description from https://docs.python.org/3/library/stdtypes.html .